envFiles/.env.rootless.devcontainer

```
@@ -0,0 +1,136 @@
+API_POSTGRES_USER=talawa
+API_REDIS_HOST=redis
+API_REDIS_PORT=6379
+API_REDIS_TEST_HOST=redis-test
+API_GRAPHQL_SCALAR_FIELD_COST=0
+API_GRAPHQL_SCALAR_RESOLVER_FIELD_COST=1
+API_GRAPHQL_OBJECT_FIELD_COST=1
+API_GRAPHQL_LIST_FIELD_COST=1
+API_GRAPHQL_NON_PAGINATED_LIST_FIELD_COST=5
+API_GRAPHQL_MUTATION_BASE_COST=10
+API_GRAPHQL_SUBSCRIPTION_BASE_COST=15
+API_RATE_LIMIT_BUCKET_CAPACITY=50000  
```

dotenv-linter (3.3.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)